azerothcore/azerothcore-wotlk
# fix(DB/spell_proc_event): The Black Heart

## CHANGES PROPOSED:
-  Fix the ICD to 50s instead of 40s of this spell https://www.wowhead.com/spell=67653/coliseum-5-tank-trinket which can be confirmed by this comment https://www.wowhead.com/item=47216/the-black-heart?bonus=6712#comments:id=799187
-  Set procFlags to 0 has it already has the correct procFlags in DBC

## TESTS PERFORMED:
- Runs




## HOW TO TEST THE CHANGES:

1. Proc the trinket and see that it will have an ICD of 50s
2. You proc it via these two procflags
PROC_FLAG_TAKEN_MELEE_HIT (Successful attack by spells that use a melee weapon)
PROC_FLAG_TAKEN_MELEE_SPELL_HIT (Damage taken from spells that use a melee weapon)


## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
